README.md

```diff
@@ -1 +1,60 @@
-# Hello_Microverse
\ No newline at end of file
+
+# Hello Microverse
+
+> It's a simple html and css project to write Hello Microverse.
+
+
+
+## Built With
+
+- html and css
+- GitHub 
+- Visual Studio Code
+
+## Live Demo (not available)
+
+
+
+## Getting Started
+
+To get a local copy up and run git clone https://github.com/Tushar151997/Hello and follow these steps.
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Open file in the VS code
+
+### Run the index.html file on any browser
+
+
+
+
+## Authors
```

Comment:
- [OPTIONAL] Kindly change this to singular  noun `Author`. This is a solo project and every solo project carries a single author.